generators/app/templates/javascript/auth/_package.json

```diff
@@ -8,13 +8,18 @@
   "engines": {
     "node": ">=10"
   },
+  "dependencies": {
+    "@verdaccio/commons-api": "^8.4.2",
+    "express": "^4.17.1"
+  },
   "devDependencies": {
-    "@babel/cli": "^7.7.0",
```

Comment:
I remove this because `"@verdaccio/babel-preset` take care of it already.

Reply:
👍 yay !!